### ISSUE TITLE
Android Gradle Plugin 7.0.0-alpha15 produces Jetifier warning

### ISSUE THREAD

**Issue by lassem** [onfido/onfido-android-sdk#158]
#### What was the expected behaviour?
After version 8.1.0 the expected behavior is that this SDK should contain no references to old support library.
Only AndroidX.

#### What happened instead?
AGP 7 produces warning that claims this SDK still has support library reference:
```
WARNING: [Processor] Library '/Users/lasse/.gradle/caches/modules-2/files-2.1/com.onfido.sdk.capture/onfido-capture-sdk-core/9.2.0/a4b31830b653feee3ed6beb2fd895b3bd546ce50/onfido-capture-sdk-core-9.2.0.aar' contains references to both AndroidX and old support library. This seems like the library is partially migrated. Jetifier will try to rewrite the library anyway.
 Example of androidX reference: 'androidx/annotation/Nullable'
 Example of support library reference: 'android/support/annotation/NonNull'
```

#### Version info:

- `onfido-capture-sdk` version: 9.2.0
- Android API version(s): NA

- `compileSdkVersion`: 30
- `targetSdkVersion`: 30
- `minSdkVersion`: 23
- Support library version (if applicable):

#### Steps to reproduce:
Compile app with jetifier turned on.
It could be that this warning is only produced in Android Studio/AGP Canary

#### Other useful info:
_screenshots, videos, etc._

**Comment by vishnus1224** [comment]
I'm trying to bundle the onfido sdk aar file as part of my final library aar file.

It's not possible to do it due to this issue as both versions of support library cannot coexist in the same project.

Any updates on when the sdk will be upgraded to AndroidX ? 

A small portion of the build output that shows classes being referenced from old support library. 
<img width="1631" alt="onfido non null annotation" src="https://user-images.githubusercontent.com/5764533/116879305-cf31a400-ac20-11eb-8ece-16efa324b8a8.png">

**Comment by gokhanaliccii** [comment]
Sorry for this problem, this issue in our backlog. I'll update when we have progress on this issue

**Comment by R4md4c** [comment]
We've fixed this issue by getting rid of traces of the old support library and will be released with the next version. 

For folks who are asking whether it's ok to disable jetifier, the answer is yes, you can ignore the warning since it's coming from the old support library annotations, and they're used during compile time rather than run-time.

**Comment by R4md4c** [comment]
Closing as the change has already been shipped as part of the [10.3.1-LTS](https://github.com/onfido/onfido-android-sdk/releases/tag/10.3.1) release
